packages/neos-ui/src/Containers/EditModePanel/Panel/index.js

```diff
@@ -32,7 +33,9 @@ export default class Panel extends PureComponent {
             speed: 300,
             slidesToShow: Math.round(modes.length / 2), // https://github.com/kenwheeler/slick/issues/1207#issuecomment-105663300
             variableWidth: true,
-            adaptiveHeight: true
+            adaptiveHeight: true,
+            nextArrow: <div><IconComponent icon="angle-right" size="big" /></div>,
+            prevArrow: <div><IconComponent icon="angle-left" size="big" /></div>
```

Comment:
This still produces an error output on the console even though it works. I could not find out how to solve that problem, because I'm not good enough with react yet. Maybe someone else can figure that out ... ```Warning: React does not recognize the `currentSlide` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `currentslide` instead. If you accidentally passed it from a parent component, remove it from the DOM element.```